codegen/templates/endpoint.tmpl

```diff
@@ -60,6 +64,22 @@ func New{{$handlerName}}(
 ) *{{$handlerName}} {
 	return &{{$handlerName}}{
 		Clients: deps.Client,
+		Logger: gateway.Logger.With(
```

Comment:
Let's avoid doing this at code generation. Let's do this at `runtime.NewRouterEndpoint()` instead.